app/models/anthology.rb

```diff
@@ -3,11 +3,10 @@ class Anthology < ActiveRecord::Base
   has_and_belongs_to_many :documents
   has_attached_file :banner,
                      styles: {
-    medium: ["3000x420>", :jpg],
-    small: ["1000x96>", :jpg],
-    thumb: ["800x45#", :jpg]
+    medium: ["1100x250>", :jpg],
```

Comment:
How does this deal with larger images of different aspect ratios? How does it deal with smaller images?

Reply:
- [ ] Wider than 1100px; scale down proportionally to 1100 by whatever. eg 
  - 2200 x 500 scales down by 50% 
  - 2200 by 2200 >> 1100 x 1100
  - 2200 by 500 >> 1100 x 250

- [ ] Narrower than 1100px?
  - Do nothing

https://github.com/thoughtbot/paperclip/wiki/Thumbnail-Generation#resizing-options